connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -7926,12 +7926,13 @@ object functions {
    * Call a builtin or temp function.
    *
    * @param funcName
-   *   function name
+   *   function name that can be qualified using the SQL syntax
    * @param cols
    *   the expression parameters of function
    * @since 3.5.0
    */
   @scala.annotation.varargs
-  def call_function(funcName: String, cols: Column*): Column = Column.fn(funcName, cols: _*)
+  def call_function(funcName: String, cols: Column*): Column =
+    Column.fn("call_function", lit(funcName) +: cols: _*)
```

Comment:
shall we add a new proto message for it? Currently it may conflict with calling a temp function named `call_function`.

Reply:
Yeah. Good suggestion.